resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -118,6 +118,8 @@ aws.settings.sam.find.title=SAM CLI Configuration
 aws.settings.sam.find.description=Select path to SAM CLI executable
 aws.settings.telemetry.option=Send anonymized usage metrics to AWS
 aws.settings.telemetry.description=Help us improve our product! Allow us to collect basic usage data.
+aws.settings.telemetry.prompt.title=AWS Toolkit
+aws.settings.telemetry.prompt.message=We'll collect anonymized usage metrics. Click <a href="">here</a> to change.
```

Comment:
Consider something like:

Anomymized usage metrics are collected by default. Click here to adjust this behavior.

Reply:
👍 - I was trying to think of better wording - this works.

Reply:
Done.